packages/client/src/views/RegisterForm/RegisterForm.tsx

```diff
@@ -314,7 +314,7 @@ function FormAppBar({
     }
     const [exitModalTitle, exitModalDescription] =
       isCorrection(declaration) ||
-      declaration.registrationStatus === SUBMISSION_STATUS.CORRECTION_REQUESTED
+      declaration.submissionStatus === SUBMISSION_STATUS.CORRECTION_REQUESTED
```

Comment:
This would probably change how the application behaves so it should rather be like this:
```suggestion
      declaration.registrationStatus === RegStatus.CORRECTION_REQUESTED
```